test/unit/slc/util.py

```diff
@@ -16,18 +17,21 @@ def __init__(
         slc_name: str,
     ):
         self.slc_name = slc_name
-        self._mock = {}
+        self._mock: dict[str, str] = {}
 
-    def get(self, key: str) -> str:
+    def get(self, key: str | CKey, default_value: str | None = None) -> str | None:
+        key = key.name if isinstance(key, CKey) else key
```

Comment:
Should we make it a function to avoid repeatitions?

Reply:
Yes, I was thinking the same. 
Please see next push.